### PR TITLE
feat: notify GitHub

### PR DESCRIPTION
**Pull request by neoziro**
See #86

screen shot 2017-02-19 at 16 48 29

### CODE REVIEW

**Comment by oliviertassinari** [comment]
Should we wait to have better authorization handling before merging it or we just sync public repositories? Also does it fix the email `null` overrides? Otherwise, we gonna broke everything.

**Comment by neoziro** [comment]
@oliviertassinari only public repositories are synced. The email null is not yet fixed, I will do it.

**Comment by neoziro** [comment]
@oliviertassinari I don't synchronize the authenticated user (we already do it at login), so I think the email problem is gone.

File: src/modules/build/notifyStatus.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    token: user.accessToken,
+  })
+
+  return github.repos.createStatus({
+    owner: owner.name,
+    repo: build.repository.name,
+    sha: build.compareScreenshotBucket.commit,
+    state,
+    description,
+    target_url: 'https://www.argos-ci.com/', // TODO compute build url
```
Comment (oliviertassinari) [inline src/modules/build/notifyStatus.js]: We can use `config.get('server.url')` to begin with.

File: src/server/models/BaseModel.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import { Model } from 'objection'
 
 export default class BaseModel extends Model {
+  constructor(props) {
+    super(props)
+    Object.assign(this, props)
```
Comment (oliviertassinari) [inline src/server/models/BaseModel.js]: Ahaha, le hack

File: src/modules/build/notifyStatus.js (hunk 1 above)
Comment (neoziro) [inline src/modules/build/notifyStatus.js]: We will change it with the url of the build, for now it's not a problem.

**Pull request merged**